modules/standard/Time.chpl

```diff
@@ -84,8 +84,13 @@ enum Day       { sunday=0, monday, tuesday, wednesday, thursday, friday, saturda
     Sunday =    6
   }
 
+  pragma "no doc"
+  proc DayOfWeek {
+    compilerError("'DayOfWeek' was renamed. Please use 'dayOfWeek' instead");
```

Comment:
Can you explain why this and `ISODayOfWeek` are implemented as compiler errors instead of warnings?

Reply:
> Can you explain why this and `ISODayOfWeek` are implemented as compiler errors instead of warnings?

https://github.com/chapel-lang/chapel/issues/18846#issuecomment-1007757555 describes a simple code that would be difficult to keep working with `DayOfWeek` deprecated and replaced. Since we don't expect either of these enums to be in wide use, it was deemed acceptable https://github.com/chapel-lang/chapel/issues/18846#issuecomment-1007799145 to make these hard errors instead.

Reply:
Ok I see, thanks for clarifying (I had thought that those comments were related to the capitalization of the enum's fields rather than deprecating/renaming the enum itself).